### PR TITLE
Page translation into multiple languages

### PR DESCRIPTION
**Pull request by rkarmuri**
Updated multiple languages on the home page. However, the page returns back to English after a word is submitted.

### CODE REVIEW

File: backend.py [hunk 1]
```diff
@@ -26,15 +26,22 @@ def homepage():
...
+    word_translated_list = logic.SQLQuery("select * from Page_Translation WHERE Language_Page = 'english';")
+    return render_template('index.html', language_list = language_list, part_of_speech_list=part_of_speech_list, sentence_List=sentence_List, error=error,page_language_list=page_language_list,word_translated_list=word_translated_list)
 
 @app.route('/Query', methods =["GET", "POST"])
 def Query():
     language_list = logic.SQLQuery("SELECT Lang_Desc FROM Lang_Ref;")
     language_selected = request.form.get('language')
     Lang_ID = logic.GetLanguageId(language_selected)
-    part_of_speech_list = logic.SQLQuery(f"SELECT Part_Desc FROM Speech_Parts WHERE Lang_ID = '{Lang_ID[0]}';")
+    part_of_speech_list = logic.SQLQuery(f"SELECT Part_Desc FROM Speech_Parts WHERE Lang_ID = '{Lang_ID}';")
```
Comment (FaizanSyed95) [inline backend.py:43]: By removing [0] from the previous code, does that mean it will give Id's for every language now?

File: backend.py [hunk 2]
```diff
@@ -26,15 +26,22 @@ def homepage():
...
     return render_template('partofspeech.html', part_of_speech_list=part_of_speech_list )
 
+@app.route('/Page', methods =["GET", "POST"])
+def Page():
```
Comment (FaizanSyed95) [inline backend.py:47]: A small description would help to understand the working of this function.

File: templates/index.html [hunk 3]
```diff
@@ -10,36 +10,37 @@
...
-    <body>
+    <body id = "wholepage">
+		<form method="POST" action="" >
 		<div id = "header">
 			<h1 id = "appname">Deelingual</h1>
-			Translate Page:
-			<select id="page_language">
-					<option>English</option>
-					<option>French</option>
+			<label id = "translate_page"for="page_language">{{word_translated_list[6]}}</label>
```
Comment (FaizanSyed95) [inline templates/index.html:17]: It looks good.

**Review by FaizanSyed95 — COMMENTED** [review]
Overall the code looks good.

File: templates/index.html (hunk 3 above)
Comment (myadlapalli) [reply templates/index.html:17]: We can add different text format for app name to make looks special from other texts from the page.

**Comment by myadlapalli** [comment]
Looks good.

**Pull request merged**